packages/mds-transactions-service/tests/service.spec.ts

```diff
@@ -105,22 +108,67 @@ describe('Transaction Service Tests', () => {
     }
   })
 
+  function* bulkTransactions(): Generator<TransactionDomainModel> {
+    let timestamp = Date.now() - 20 * 1000
+    for (let i = 0; i < 20; i++) {
+      yield {
+        transaction_id: uuid(),
+        provider_id,
+        device_id,
+        timestamp,
+        amount: 100, // "I'd buy THAT for a dollar!"
+        fee_type: 'base_fee',
+        receipt
+      }
+      timestamp += 1000
+    }
+  }
+
+  it('Bulk Post 20 Transactions', async () => {
+    const transactions = await TransactionServiceClient.createTransactions([...bulkTransactions()])
+    expect(transactions[0].device_id).toEqual(device_id)
+    // TODO more checks
+  })
+
   it('Get All Transactions', async () => {
```

Comment:
Some good tests, but you could probably add more to exercise  the other params in `TransactionSearchParams`